src/app/(content-pages)/transform-property/TransformPropertyPage.tsx

```diff
@@ -275,139 +286,126 @@ export default function TransformPropertyPage() {
-                and community groups. Small grants and donations might be easier
-                to get for groups like yours. And don't forget about raising
-                money right in the community through events or online campaigns.
-              </div>
-            </AccordionItem>
+        <div className="bg-gray-100 rounded-lg p-8 mt-8">
+          <h3 className="heading-xl font-bold mb-4">Maintenance</h3>
+          <p className="body-md mb-4">
+            Transforming a vacant property isn’t a one-and-done event. In fact,
+            many properties that are currently vacant were transformed years
+            ago—but not maintained in the decades since. In order to make sure
+            that your intervention is high-impact and long-lived, make sure to
+            have a maintenance plan in place.
+          </p>
+          <p className="body-md mb-4">
+            Consider who will be responsible for maintaining the property, what
+            their tasks will be, what kind of resources they will need, and how
+            you will fund this ongoing maintenance. Often, partnering with an
+            established organization like a CDC can help make sure that you have
+            the capacity and resources needed to ensure that your intervention
+            continues to be effective for many years into the future.
+          </p>
+        </div>
 
-            <AccordionItem
-              key="3"
-              aria-label="Gentrification"
-              title="Gentrification"
-              subtitle="Click to expand"
-              className="font-bold text-large"
-            >
-              <div className="text-base font-normal">
-                Remember, your projects shouldn't push out the current
-                residents. Keep them at the heart of what you do, and keep an
-                eye on how things change in the neighborhood. Work together with
-                other local groups, schools, and businesses - it's a great way
-                to get more resources and ideas. Have regular meetings with
-                people in the community to keep them in the loop and involved in
-                decisions.
-              </div>
-            </AccordionItem>
-          </Accordion>
+        <div className="bg-gray-100 rounded-lg p-8 mt-8">
+          <h3 className="heading-xl font-bold mb-4">Gentrification </h3>
+          <p className="body-md">
+            Our goal is to help community groups reduce violence through
+            cleaning and greening methods in their neighborhoods. We recognize
+            that may inadvertently increase property values, increase rental
+            prices and displace already-marginalized Philadelphians. We are
+            working to avoid this by meeting extensively with community groups
+            and responding to their concerns. Read more about this topic.
+            {/* Link to gentrification page */}
```

Comment:
If we don't actually have a link yet, I think we can remove the "Read more..." copy. @thansidwell thoughts?